simulation/revolve2/simulation/scene/geometry/textures/_texture.py

```diff
@@ -0,0 +1,118 @@
+from abc import ABC
+from dataclasses import dataclass
+
+from ..._color import Color
+
+
+@dataclass
+class Texture(ABC):
+    """An abstract texture for mujoco models."""
```

Comment:
You cannot mention mujoco or make anything specific for mujoco. this is an abstraction layer for physics simulators